NRLMMD-GEOIPS/geoips
# 74 draft classes for yaml based plugins

# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#74


# Testing Instructions

- Uninstall all geoips packages (e.g. geoips, data_fusion, geoips_clavrx)
- Reinstall geoips with `pip install .` (no `-e`).
- Run the following tests:
  - `./tests/test_full_install.sh`
  - `pytest tests/test_pytest`

Then run this test:
```
from geoips import interfaces
for name in interfaces.__all__:
    inter = getattr(interfaces, name)
    print(name, len(inter.get_plugins()))
```
which should return something like this with no errors:
```
algorithms 10
colormaps 20
filename_formatters 9
interpolators 3
output_formatters 12
procflows 2
readers 28
title_formatters 3
products 26
product_defaults 75
feature_annotators 4
gridline_annotators 6
```

# Summary


# Output
